source/backend/plugin/CarlaPluginFluidSynth.cpp

```diff
@@ -1683,13 +1770,19 @@ class CarlaPluginFluidSynth : public CarlaPlugin
     float** fAudio16Buffers;
     float   fParamBuffers[FluidSynthParametersMax];
 
+    static bool  sbFluidDefaultsStored;
```

Comment:
I meant before to use "s" prefix *instead* of "b" or "f", just a single one.
Rest looks quite fine, thanks!